pkg/provider/cloud/vsphere/provider_test.go

```diff
@@ -28,6 +28,10 @@ import (
 	"k8c.io/kubermatic/v2/pkg/resources"
 )
 
+const (
+	fakeStoragePolicy = "fake_storage_policy"
+)
+
 func TestGetCredentialsForCluster(t *testing.T) {
```

Comment:
Do we have any validation for the specified StoragePolicy? For example, I see that we validate and ensure that the specified Datastore actually exists. Should we do that for the StoragePolicy as well?

Reply:
I'll check it out